.github/workflows/test_template.yaml

```
@@ -24,6 +34,9 @@ jobs:
             linux-386-unit-1-cpu)
               GOARCH=386 PASSES='unit' RACE='false' CPU='1' ./scripts/test.sh -p=4
               ;;
+            linux-arm64-unit-test-4-cpu-race)
```

This new target is named slightly differently, let's keep them consistent:

```suggestion
            linux-arm64-unit-4-cpu-race)
```